### ISSUE TITLE
 Review the alt text for Citizen Engagement page images and ensure that they adhere to WCAG

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2570]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Citizen Engagement page.

#### Details
Developers identified that some alt texts in the Citizen Engagement page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on to the Resources/Instructions section of this issue at the bottom where it says "Google sheet Tab for your page".
- [x] For each image on the Citizen Engagement page identify if the alt text needs to be changed, or not, using the recourses below, and fill out the google sheet. To locate the alt text use the Google Chrome browser, right click "inspect" to view pages code.
- [x] After completing auditing each image change your google sheet tabs [Status] to Audit Complete.
- [x] Review with Design Lead or another member of the team
- [x] Mark spreadsheet tab as QA (e.g., About Page - Audit QA)
- [x] Add note at bottom of issue saying its ready for QA and use the close with comment, so that it goes to the QA column

Product
- [x] Review the work and sign off on it if ready
   - [x] Mark spreadsheet tab as Complete (e.g., About Page - QA Complete)
   - [x] Add Ready for dev leads label
   - [x] Move to New Issue Approval Column

Development
- [x] Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix
- [x] Close this issue

#### Summary
If images provide more meaning = needs alt text
If images are decorative = null alt text (alt="")

### Resources/Instructions
[Citizen Engagement page](https://www.hackforla.org/citizen-engagement)
[Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
[Decorative Images and alt text](https://www.w3.org/WAI/tutorials/images/decorative/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=865455753)

**Comment by jenjen26** [comment]
Blockers: None
ETA 2/24/22

**Comment by SAUMILDHANKAR** [comment]
Issues have been created and are being tracked in #3089, so moving this issue to done column.

**Comment by JessicaLucindaCheng** [comment]
@SAUMILDHANKAR I see you have commented https://github.com/hackforla/website/issues/2570#issuecomment-1116975337 that you already wrote the issue. After you commented with the issue(s) you made, please remove the `ready for dev leads` label so we don't spend extra time checking issues that already have been followed up by a dev lead. 

I will remove the label this time.
